### DISCUSSION TITLE
Sveltejs support

### DISCUSSION THREAD

**Discussion by ansarizafar** [concrete-utopia/utopia#1405] [Q&A]
Is there any plan to provide support for creating applications with sveltejs.

**Comment by maltenuhn** [comment]
Hi there - unfortunately we don't have immediate plans to support svelte. We absolutely do want to support other frameworks and libraries. and svelte / vue are the most frequently mentioned ones. But it's a non-trivial effort, and we feel we'll get there faster by first improving and tightening the React loop, then abstract it out.
